app/src/main/java/com/lucasfagundes/androidudemycourse/feature/learn_english/LearnEnglishMainFragment.kt

```diff
@@ -1,30 +1,44 @@
 package com.lucasfagundes.androidudemycourse.feature.learn_english
 
 import android.os.Bundle
+import android.view.*
+import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.appcompat.app.AppCompatActivity
+import com.lucasfagundes.androidudemycourse.R
+import com.lucasfagundes.androidudemycourse.databinding.ActivityMainBinding
 import com.lucasfagundes.androidudemycourse.databinding.FragmentLearnEnglishMainBinding
+import com.lucasfagundes.androidudemycourse.feature.learn_english.adapter.LearnEnglishPagerAdapter
 
 class LearnEnglishMainFragment : Fragment() {
 
-    private lateinit var binding : FragmentLearnEnglishMainBinding
+    private lateinit var binding: FragmentLearnEnglishMainBinding
+    private lateinit var activityBinding : ActivityMainBinding
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-        (activity as AppCompatActivity).supportActionBar?.hide()
-
-        binding = FragmentLearnEnglishMainBinding.inflate(inflater,container,false)
+        binding = FragmentLearnEnglishMainBinding.inflate(inflater, container, false)
         return binding.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding.viewPager.adapter = LearnEnglishPagerAdapter(childFragmentManager)
         binding.tabLayout.setupWithViewPager(binding.viewPager)
 
+        binding.viewPager.clearOnPageChangeListeners()
+        changeStatusBarColor()
+
+        //activityBinding.appBarMain.mainToolbar.setBackgroundColor(R.color.brown)
```

Comment:
se esse trecho nao for utilizado seria melhor apagar pra deixar o codigo mais limpo